### ISSUE TITLE
Rename this repo to jhalfs

### ISSUE THREAD

**Issue by jhuntwork** [automate-lfs/jhalfs#2]
This is just to test out the issue system.

And also see if anyone else prefers lowercase.

**Comment by SheepReaper** [comment]
I suppose it depends on where you stand in the whole abbreviation vs
acronym debate.

I don't think it's an abbreviation since it's all initials of a multiple
word phrase. But it's also not an acronym since it's not really
pronounceable.

Unless....
Jay-Halfs?
Jay-Halves?

I don't think you can apply camel case because it would be way too long. If
you shorten it to initials, it's all caps anyways (And resharper flags it
if it's all caps) It suggests Jhalfs instead, but yuck....

Maybe do an Apple: jHALFS  or jhALFS or drop the h and do jALFS



...



Just testing how much sarcasm you can handle...



...

It really doesn't bother me either way, but if I must vote, I don't like
yelling, so all lower-case is my vote.

**Comment by jhuntwork** [comment]
Yeah. Those are all good comments. That's also partly why I made the class name in the python module JHalfs. But they are also comments for a conversation that happened years ago. 🤓

My take: If we're keeping the name 'jhalfs', make it lowercase like it's been for years in svn. But if we're going forward to switching to Python or other major overhauls and we want to come up with a new name at the same time... well that's a new conversation.

Probably good to get feedback from @pierre-labastie

**Comment by SheepReaper** [comment]
Here's a spanner for this: I'll volunteer to write a third equivalent.
ps-jhalfs or jhalfs-ps.
If the core/original project was jhalfs shell scripts
and the natural evolution is jhalfs python
then natually, jhalfs PowerShell Core comes in the pattern...

I'm partially serious.

PS Core is cross platform, and I'm curious what equivalent scripts would
look like. If I get some spare time, I'll look into the feasibility of this
(similar effort to pythonizing).

Oh, and I can python too... @pierre-labastie
<https://github.com/pierre-labastie> you'll enjoy python. My new work
(prospective) asked if I could learn Flask, I said sure... Then I realized
I needed to learn python... (Mind I've been a .NET developer for a while
now, can C# proficiently, and javascript for web stuff) I've been avoiding
python for the longest time. Only to realize that it's the first language
many schools teach now, and it's super popular for general purpose because
it's easy to pick up. I spent 3 hours learning python, and now i'm on to
Flask.

Here's my badge of office:
[image: image.png][image: image.png]
[Profile](
https://www.linkedin.com/in/bryan-gonzalez-2b86ba67/detail/recent-activity/)
